### PR TITLE
Problem 17: Test Case

### PR DESCRIPTION
**Pull request by seemcat**

### CODE REVIEW

File: solutions/17.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const solution = (arrOfarr) => {
+  const firstElement = arrOfarr[0].length;
+  lengthOfArr = arrOfarr.length;
+  let i = 1;
+
+  while (i < lengthOfArr){
+    if (arrOfarr[i].length === firstElement){
```
Comment (songz) [inline solutions/17.js]: Inside while loop:
```
if (arrOfarr[i].length !== firstLength) {
  return false;
}
i ++;
```

File: test/17.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const expect = require('chai').expect;
+let solution = require('../solutions/17').solution;
+// solution = require('./yourSolution').solution;
+
+describe('squareMatrix', () => {
+  it('should return true', () => {
```
Comment (songz) [inline test/17.js]: Please use better messaging. `should be true when input is 3x3`, `should be false when input is 2x3`, `should be true when input is 2x2`

**Comment by songz** [comment]
Please fix.

File: test/17.js [hunk 3]
```diff
@@ -1,6 +1,5 @@
 const expect = require('chai').expect;
 let solution = require('../solutions/17').solution;
```
Comment (songz) [inline test/17.js]: since there are now 2 solutions, please test both.

File: solutions/17.js [hunk 4]
```diff
@@ -1,7 +1,21 @@
...
 // Square 2D Array
+// Maricris Bonzo
+
+const solution1 = (arrOfarr) => {
+  const firstIndexLength = arrOfarr[0].length;
+  lengthOfArr = arrOfarr.length;
+  let i = 1;
 
-const solution = (arr) => {
+  while (i < lengthOfArr){
```
Comment (yjlim5) [inline solutions/17.js]: remove line 6 and directly call `.length` function inside this condition.

**Pull request closed without merging**